docs/features/LOGGING.md

```diff
@@ -56,3 +58,50 @@ output.elasticsearch:
 If you are running Crawler inside of a Docker container and you have set it up to write logs to a mounted volume, you can simply point the `paths` fields to the directory your logs are being written to.
 
 Check out [Filebeat's quickstart guide](https://www.elastic.co/guide/en/beats/filebeat/current/index.html) to learn more about how to get it installed and running.
+
+## Event Logs
+### What are event logs?
+Event logs are detailed logs that contain information regarding pertinent 'events' Crawler undertakes during normal operation. These events include, but are not limited to:
```

Comment:
Do we know what every event is? If so, let's include all of them in the list below.